modules/ROOT/pages/to-request-access.adoc

```diff
@@ -26,19 +26,19 @@ If terms and conditions are set for an asset, you are prompted to accept the
 terms before access is granted, and the created contract includes the terms and
 conditions.
 
-. Click an API asset to view more details.
+. Click an API or API Group asset to view more details.
 . Click *Request Access*.
-. Choose the API instance from the list.
+. Choose the API instance or API Group instance from the list.
 . If you are using an existing application, choose from the list of applications.
 +
-An application and its API instance use the same client provider. If an organization administrator has configured xref:api-manager::configure-multiple-credential-providers.adoc[multiple client providers] in Anypoint API Manager, then the application list shows only the applications that use the same client provider as the selected API instance.
+An application and its asset instance use the same client provider. If an organization administrator has configured xref:api-manager::configure-multiple-credential-providers.adoc[multiple client providers] in Anypoint API Manager, then the application list shows only the applications that use the same client provider as the selected asset instance.
```

Comment:
`asset instance` is a term we don't use. We go with "asset" for the thing that lives in Exchange, or "instance" for the thing that lives in API Manager and has contracts. (The same way an "exchange-asset" has a type, the "api-manager-instance" can be of type "api" or "api-group")

Reply:
That makes perfect sense! Updated to fix this.